src/common/snippets/src/op/rank_normalization.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/op/rank_normalization.hpp"
+
+namespace ov {
+namespace snippets {
+namespace op {
+
+RankNormalization::RankNormalization(const Output<Node>& data, size_t num_prepend, size_t num_append) :
+    Op({data}), m_num_prepend(num_prepend), m_num_append(num_append) {
+    constructor_validate_and_infer_types();
+}
+
+
+std::shared_ptr<ov::Node> RankNormalization::clone_with_new_inputs(const OutputVector& new_args) const {
+    check_new_args_count(this, new_args);
+    return std::make_shared<RankNormalization>(new_args[0], m_num_prepend, m_num_append);
+}
+
+void RankNormalization::validate_and_infer_types() {
+    auto new_shape = get_input_partial_shape(0);
+    // Note: other values are not allowed, only planar + blocked layout combination can be normalized.
+    NODE_VALIDATION_CHECK(this, m_num_append == 0 || m_num_append == 1,
+                          "num_append could be only 0 or 1, other values are not allowed.");
```

Comment:
```suggestion
    NODE_VALIDATION_CHECK(this, ov::snppets::utils::one_of(m_num_append, 0, 1),
                          "num_append could be only 0 or 1, other values are not allowed.");
```

Reply:
👍

Reply:
Changed as you suggested and the build fails with 

> error: comparison of integer expressions of different signedness: 'long unsigned int' and 'int'

I know, I can write `0lu` and `1lu`, but do I have to think about such things all the time? Good-old comparison operators would do type all the necessary casting for me. Anyway, that's why I don't use tiny helper functions - I don't trust them 🥲